### ISSUE TITLE
package.json points to nonexistent index.js file

### ISSUE THREAD

**Issue by zcorpan** [saucelabs/cddl#4]
Trying to run the sample code in the README as a node script, I get this error:

```
% node test.js

internal/modules/cjs/loader.js:311
      throw err;
      ^

Error: Cannot find module '<path>/node_modules/cddl/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:303:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:516:18)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:867:27)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (<path>/test.js:1:14)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32) {
  code: 'MODULE_NOT_FOUND',
  path: '<path>/node_modules/cddl/package.json',
  requestPath: 'cddl'
}
```

The `package.json` file in this repo says
```
  "main": "index.js",
```
But there is no `index.js` file.

Am I holding it wrong, or is it a bug?

**Comment by christian-bromann** [comment]
@zcorpan thanks for raising the issue.

This is indeed a bug, the `package.json` should say:
```json
"main": "build/index.js",
```

**Comment by zcorpan** [comment]
This seems to work:
```js
const CDDL = require('cddl').default

const ast = CDDL.parse('./spec.cddl')
console.log(ast)
```

**Comment by diemol** [comment]
Closed via #5
